app/routines/find_or_create_unclaimed_user.rb

```diff
@@ -48,8 +48,9 @@ def create_user(options)
       # they'll be prompted to reset it
       identity.password_expires_at = DateTime.now
       identity.save!
-      user.authentications.create!(   # TODO review this creation of authentication (otherwise only in SessionsCallback)
-        provider: 'identity', uid: identity.id.to_s,
+      user.authentications.create!(
+        # TODO review this creation of authentication (otherwise only in SessionsCreate)
+        provider: 'identity', uid: identity.id.to_s
```

Comment:
Kind of thought unclaimed user stuff got left behind in the big Accounts revamp we did back in April, and thought I left a huge comment to that effect, but not seeing it now.